public/app/core/components/Login/LoginPage.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { FC } from 'react';
+import config from 'app/core/config';
+import { UserSignup } from './UserSignup';
+import { LoginServiceButtons } from './LoginServiceButtons';
+import LoginCtrl from './LoginCtrl';
+import { LoginForm } from './LoginForm';
+import { ChangePassword } from './ChangePassword';
+import { CSSTransition } from 'react-transition-group';
+
+const isOauthEnabled = () => Object.keys(config.oauth).length > 0;
+
+export const LoginPage: FC = () => {
+  return (
+    <div className="login container">
+      <div className="login-content">
+        <div className="login-branding">
+          <img className="logo-icon" src="public/img/grafana_icon.svg" alt="Grafana" />
+          <div className="logo-wordmark" />
+        </div>
+        <LoginCtrl>
+          {({ login, isLoggingIn, changePassword, toGrafana, isChangingPassword }) => (
+            <div className="login-out-box">
+              <div className={`login-inner-box ${isChangingPassword ? 'hidden' : ''}`} id="login-view">
+                <LoginForm
+                  displayLoginFields={!config.disableLoginForm}
+                  displayForgotPassword={!(config.ldapEnabled || config.authProxyEnabled)}
```

Comment:
Since this is sort of logic i think it belongs to the LoginCtrl, so the dependency on the app config should belong there too.